ckanext/harvest/logic/action/update.py

```diff
@@ -428,18 +448,23 @@ def harvest_source_reindex(context, data_dict):
     if 'extras_as_string'in context:
         del context['extras_as_string']
     context.update({'ignore_auth': True})
-    package_dict = logic.get_action('harvest_source_show')(context,
-        {'id': harvest_source_id})
-    log.debug('Updating search index for harvest source {0}'.format(harvest_source_id))
+    package_dict = logic.get_action('harvest_source_show')(
+        context, {'id': harvest_source_id})
+    log.debug('Updating search index for harvest source {0}'.format(
+        harvest_source_id))
 
     # Remove configuration values
     new_dict = {}
-    if package_dict.get('config'):
+    if package_dict.get('config', None):
```

Comment:
I'm not sure adding `, None` adds anything here since that is the default - perhaps you might remove it? There are some others too that would be usefully removed.